code/src/reducers/sizes.js

```diff
@@ -0,0 +1,22 @@
+import { createSlice } from '@reduxjs/toolkit';
+
+// Store to solve resizing based on window size
+// smallest screen is 320px
+const myInitialState = {
+	screenSize: 300
+};
+export const size = createSlice({
+	name: 'size',
+	initialState: myInitialState,
+	reducers: {
+		changeSize: (state, action) => {
+			if (action.payload >= 860) {
+				state.screenSize = 800;
+			} else if (action.payload > 300) {
+				state.screenSize = action.payload * 0.95;
+			} else {
+				state.screenSize = 300;
+			}
+		}
```

Comment:
Are you using this instead of media queries?
The website is not really responsive for me though...
Although that was not a requirement for this project but if you have some time, that would be a good thing to improve.